runtime/doc/eval.txt

```diff
@@ -7989,6 +7989,24 @@ readdirex({directory} [, {expr}])			*readdirex()*
 		When {expr} is a function the entry is passed as the argument.
 		For example, to get a list of files ending in ".txt": >
 		  readdirex(dirname, {e -> e.name =~ '.txt$'})
+<
+		The optional {dict} argument allows for further custom
+		values. Currently this is used to specify if and how sorting
+		should be performed. The dict can have the following members:
+
+		    sort    How to sort the result returned from the system.
+			    Valid values are:
+				none	do not sort (fastest method)
+				case	sort case senstive
+				icase	sort case insensitive
+				collate sort using collation order of your
+					|locale|
+			    Other values are silently ignored.
+
+		For example, to get a list of alle files in the current
```

Comment:
Typo: "all", not "alle".  Also, one line below: "entries", not "entires".

Reply:
thanks fixed.